js/player.js

```diff
@@ -123,4 +141,25 @@ function Player() {
         }
       });
     };
+    this.hasDiscoveredExactly = function(x){
+		if (seenTiles.length === x){
+			return true;
+		}else{
+			return false;
+		}
+	};
+	this.hasDiscoveredAtLeast = function(x){
+		if (seenTiles.length >= x){
+			return true;
+		}else{
+			return false;
+		}
+	};
+	this.hasDiscoveredLessThan = function(x){
+		if (seenTiles.length < x){
+			return true;
+		}else{
+			return false;
+		}
+	};
```

Comment:
I think we can consolidate these 3 functions into one which simply gives the number of tiles visited

Reply:
Sure thing